documentation/docs/02-layouts.md

```diff
@@ -113,64 +113,60 @@ src/routes/
 └ __layout-foo.svelte
 ```
 
-#### Resetting layouts
+#### Inheritance chains
 
-Any page, at any depth, can reset to the home layout, ignoring any intermediate `__layout` components by referencing the special `~` layout:
+Layouts can themselves choose which other layouts they inherit from. Ordinarily, they will inherit from any default layouts 'above' them in the tree, but they can inherit named layouts using the same logic as pages. For example, `a/b/c/page.svelte` will inherit from `__layout-foo.svelte` and `a/b/c/__layout@foo.svelte` but _not_ `__layout.svelte` or `a/b/__layout.svelte`.
 
 ```
 src/routes/
 ├ a/
 │ ├ b/
 │ │ ├ c/
-│ │ │ └ index@~.svelte
+│ │ │ ├ __layout@foo.svelte
+│ │ │ └ page.svelte
 │ │ └ __layout.svelte
-│ └ __layout.svelte
-└ __layout.svelte
+│ ├ __layout.svelte
+│ └ __layout-foo.svelte
 ```
 
-To reset _all_ layouts, including the home layout, a page can specify no layout, like this `index@.svelte` component:
+Layouts will inherit from 'sibling' layouts as well as parents — you'll never need to do this, but you _could_:
 
 ```
 src/routes/
-├ a/
-│ ├ b/
-│ │ ├ c/
-│ │ │ └ index@.svelte
-│ │ └ __layout.svelte
-│ └ __layout.svelte
-└ __layout.svelte
+├ __layout.svelte
+├ __layout-a@.svelte
+├ __layout-b@a.svelte
+├ __layout-c@b.svelte
+└ nested@c.svelte
 ```
 
-#### Inheritance chains
+Here, the inheritance chain would look like this (note that `__layout-a@` points at the unnamed default layout, because no name is specified):
 
-Layouts can themselves choose which other layouts they inherit from. Ordinarily, they will inherit from any default layouts 'above' them in the tree, but they can inherit named layouts using the same logic as pages. For example, `a/b/c/page.svelte` will inherit from `__layout-foo.svelte` and `a/b/c/__layout@foo.svelte` but _not_ `__layout.svelte` or `a/b/__layout.svelte`.
+```
+__layout.svelte
+  __layout-a@.svelte
+    __layout-b@a.svelte
+      __layout-c@b.svelte
+        nested@c.svelte
+```
+
+#### Resetting layouts
+
+Sometimes you need a page to use the 'home' layout, regardless of its URL. You can do this by creating a top-level named layout that inherits from the default layout by using `@` without a name:
```

Comment:
```suggestion
You can create a named layout that does not inherit from others by using `@` without a name. E.g. you could do this to have a page use the 'home' layout, regardless of its URL:
```

Reply:
marking this resolved as the API has changed since this suggestion